README.md

```diff
@@ -1,4 +1,4 @@
-# How to use the unified workflow for the coupled ufs-weather-model (work in progress)
+# How to use the unified workflow for the coupled ufs-aerosol app (work in progress)
```

Comment:
??? Feature/coupled-crow is now being changed to the aerosol app?

Reply:
Change to: 

How to use the unified (or feature/coupled-crow) workflow for the coupled ufs-weather-model or the aersol app